src/client/src/pages/detail/labeling/labelingPanel.tsx

```diff
@@ -238,13 +246,25 @@ const LabelingPanel: FC<LabelingPanelProps> = ({ boreholeId }) => {
               sx={{ height: "44px" }}
             />
           </Stack>
-          <ZoomControls onZoomIn={zoomIn} onZoomOut={zoomOut} onFitToExtent={fitToExtent} />
+          <Stack
+            sx={{
+              position: "absolute",
+              top: theme.spacing(2),
+              right: theme.spacing(2),
+              zIndex: "500",
+              gap: 1,
+            }}>
+            <ZoomControls onZoomIn={zoomIn} onZoomOut={zoomOut} onFitToExtent={fitToExtent} applyBaseStyling={false} />
```

Comment:
PP: Ich hätte glaube ich den Rotation Button auch in die Controls genommen (dann zb "MapControls" ) und den mit einem Flag ausgeblendet, statt den Basestyle.